src/components/academy/grading/index.tsx

```diff
@@ -25,6 +34,7 @@ type State = {
   columnDefs: ColDef[];
   filterValue: string;
   groupFilterEnabled: boolean;
+  unsubmitAlert: JSX.Element | null;
```

Comment:
Why is the `unsubmitAlert` part of the state?

Reply:
Removed, isolated the unsubmit cell component